hnn_core/tests/test_optimize_evoked.py

```diff
@@ -0,0 +1,84 @@
+# Authors: Carolina Fernandez <cxf418@miami.edu>
```

Comment:
The directory structure of your tests should reflect the directory structure of your module. At the very least, I would say, the names should be consistent. If your original file is `optimizer.py` then the test file should be called `test_optimizer.py` not `test_optimize_evoked.py`